ext/FHistHDF5Ext.jl

```diff
@@ -28,7 +28,7 @@ function h5writehist(filename::AbstractString, path::AbstractString, h::Union{Hi
         for (dim, edges) in enumerate(h.hist.edges)
             write(g, "edges_$dim", collect(edges))
         end
-        attributes(g)["overflow"] = string(h.overflow)
+        attributes(g)["overflow"] = h.overflow
         attributes(g)["nentries"] = h.nentries.x
```

Comment:
what is `.x` I don't even remember

Reply:
It is a value unwrap from a `Base.RefValue` or so... (I also cannot recall but will do tomorrow ;)

Reply:
Ah you are right. I did really worked with those yet.

Reply:
Fixed...

Reply:
Well, the correct term is "dereference" of course 😆